modules/@apostrophecms/piece-type/ui/apos/components/AposPiecesManager.vue

```diff
@@ -284,6 +289,25 @@ export default {
       this.currentPage = 1;
 
       this.getPieces();
+    },
+
+    shortcutNew(event) {
+      const interesting = (event.keyCode === 78 || event.keyCode === 67); // C(reate) or N(ew)
```

Comment:
We need the keyboard shortcut to show the keyboard shortcuts. I imagine this will be useful to wrap up in shared code later.